src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -903,11 +828,9 @@ struct GhValenciaDivCleanTemplateBase<
       tmpl::conditional_t<
           use_dg_subcell,
           tmpl::list<evolution::dg::subcell::Actions::SetSubcellGrid<
-              volume_dim, system, use_numeric_initial_data>>,
-          tmpl::conditional_t<use_numeric_initial_data, tmpl::list<>,
-                              evolution::Initialization::Actions::SetVariables<
-                                  ::domain::Tags::Coordinates<
-                                      volume_dim, Frame::ElementLogical>>>>,
+              volume_dim, system, true>>,
+          evolution::Initialization::Actions::SetVariables<
```

Comment:
On mobile now so can't check this easily: does this still work with numeric ID? Before this action was just skipped.

Reply:
I think this conditional branch is never tested and probably doesn't work, because `use_dg_subcell` is always toggled on. Maybe add a comment that it hasn't been tested?